llvm/lib/Support/Unix/Process.inc

```diff
@@ -252,13 +257,21 @@ std::error_code Process::FixupStandardFileDescriptors() {
 
     if (NullFD == StandardFD)
       FDC.keepOpen();
+#if defined(__wasi__)
+    else // WASI does not have `dup2` by design. Return EBADF.
```

Comment:
Does wasi have /dev/null? If not, this whole function is pointless. If so, that could be opened directly, instead. Actually, given the number of changes here for wasi, shouldn't wasi just have its own llvm/lib/Support/WASI/Process.inc file?

Reply:
No, no `/dev/null`.